locked-asset/energy-factory/src/lib.rs

```diff
@@ -161,13 +165,23 @@ pub trait SimpleLockEnergy:
 
         self.send()
             .esdt_local_mint(&output_payment.token_identifier, 0, &output_payment.amount);
-        self.send().direct_esdt(
-            &caller,
-            &output_payment.token_identifier,
+
+        let token_payment = EsdtTokenPayment::new(
+            output_payment.token_identifier.clone(),
             0,
-            &output_payment.amount,
+            output_payment.amount.clone(),
         );
 
+        let unbond_epochs = self.unbond_epochs().get();
```

Comment:
we do not need to lock for unbonding if FEEs are 0.

Reply:
Updated.